exchange/bidder.go

```diff
@@ -134,36 +134,25 @@ func (bidder *bidderAdapter) requestBid(ctx context.Context, request *openrtb.Bi
 			if bidResponse != nil {
 
 				if bidResponse.Currency == "" {
-					bidResponse.Currency = "USD"
-				}
-
-				// Related to #281 - currency support
-				// Prebid can't make sure that each HTTP call returns bids with the same currency as the others.
-				// If a Bidder makes two HTTP calls, and their servers respond with different currencies,
-				// we will consider the first call currency as standard currency and then reject others which contradict it.
-				if firstHTTPCallCurrency == "" { // First HTTP call
-					firstHTTPCallCurrency = bidResponse.Currency
+					// Empty currency means default currency `USD`
+					bidResponse.Currency = defaultCurrency
 				}
 
-				// TODO: #281 - Once currencies rate conversion is out, this shouldn't be an issue anymore, we will only
-				// need to convert the bid price based on the currency.
-				if firstHTTPCallCurrency == bidResponse.Currency {
+				// Try to get a conversion rate
+				if conversionRate, err := conversions.GetRate(bidResponse.Currency, "USD"); err == nil {
+					// Conversion rate found, using it for conversion
```

Comment:
Please leave a `TODO: #280` comment somewhere around here.

Before this feature is closed, this code should try to convert every to element of `request.cur`, and use the first one which succeeds. No need to do that in this PR... but I don't want to lose track of it.

Reply:
Yes, just added ! I would prefer put it in another PR (the one doing all the `cur` mapping so it will be cleared maybe.